### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by vechicin**
Created a desktop version for my portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,23 +239,38 @@ <h2 class="contact-title">Get started</h2>
...
                 </div>
+                <label class="textarea-label" for="user-message">Message</label>
                 <textarea name="user-message" id="user-message" cols="30" rows="10" maxlength="500"
                     placeholder="Enter your message here..."></textarea>
                 <button type="submit" class="submit-button">Start collaboration</button>
+                <div class="desktop-form-image">
+                    <img class="geo-image" src="./images/image-geometry_7.png"
+                        alt="This should be a colorful, geometric design">
+                </div>
             </form>
```
Comment (omar-labana) [inline index.html:268]: - [ ] On the desktop screen, the form is not properly positioned, kindly inspect your media query to fix this issue.
image

File: index.html [hunk 2]
```diff
@@ -16,37 +16,101 @@
...
+                        <h3 class="project-name">Project name goes here</h3>
+                        <ul class="project-tags">
+                            <li class="languages">HTML/CSS</li>
+                            <li class="languages">Ruby on Rails</li>
+                            <li class="languages">JavaScript</li>
+                        </ul>
+                        <button type="button" class="project-button">
+                            See this project <span class="arrow-icon"></span>
+                        </button>
+                    </div>
```
Comment (omar-labana) [inline index.html:69]: - [ ] Your hover element styling is not quite similar to the Figma design, kindly adjust your styling according to the Figma design.
- Figma design:
image
- Your styling:
image

File: index.html [hunk 3]
```diff
@@ -16,37 +16,101 @@
...
+                    and contact me.</p>
+                <button type="button" class="actual-button">
+                    Start collaboration
+                </button>
+            </div>
+            <div class="side-image"></div>
         </section>
+        <div class="bottom-image"></div>
         <section class="projects-section">
-            <h2 class="projects-title">Projects</h2>
```
Comment (omar-labana) [inline index.html:35]: - [ ] On the desktop screen, your projects section is causing the page to scroll horizontally, please inspect your stylings and look for what is causing this scrolling behavior.

- [ ] On the mobile screen, your projects section and/or the next section also has the same scrolling behavior, Kindly inspect this as well.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @vechicin, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
Your repo is professional and there are no linter errors. ✔️ 
Your repo is following GitHub flow. ✔️
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by eri8-9 — APPROVED** [review]
Hi @vechicin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did most of the work where the last code reviewer was pointing, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**